src/jit/compiler.hpp

```diff
@@ -1289,6 +1289,10 @@ inline void GenTree::SetOper(genTreeOps oper, ValueNumberUpdate vnUpdate)
 
     assert(GenTree::s_gtNodeSizes[oper] == TREE_NODE_SZ_SMALL || (gtDebugFlags & GTF_DEBUG_NODE_LARGE));
 
+#if NODEBASH_STATS
+    RecordOperBashing(OperGet(), oper);
```

Comment:
Ah, yes, my schizophrenia showing - I couldn't decide which was cleaner / less ugly so I used both; sorry about that. Any thoughts as far as which is better? The #if/#ifdef frequency is pretty high already but making calls disappear through macros or empty inline's isn't exactly the cleanest thing either. Opinions?